pkgs/by-name/on/onthespot/package.nix

```diff
@@ -4,22 +4,26 @@
   fetchFromGitHub,
   makeDesktopItem,
   python3,
-  libsForQt5,
+  kdePackages,
   ffmpeg,
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "onthespot";
-  version = "0.5";
+  version = "1.0.2";
   pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "casualsnek";
+    owner = "justin025";
     repo = "onthespot";
     rev = "refs/tags/v${version}";
-    hash = "sha256-VaJBNsT7uNOGY43GnzhUqDQNiPoFZcc2UaIfOKgkufg=";
+    hash = "sha256-VuzFsmMqnHfDNuqRXjL3DoZ70XqgRLICv46eh3ySVtA=";
   };
 
+  build-system = [ python3.pkgs.setuptools ];
+
+  env.PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION = "python";
```

Comment:
I'm not sure why this doesn't work. There seems to be an issue with `protobuf` and setting this as a workaround should work.